Irina/src/main/java/oop/Company.java

```diff
@@ -0,0 +1,14 @@
+package oop;
+
+public class Company extends AbstractUser {
+    private final String companyName;
```

Comment:
Same here, you already have name field in parent class

Reply:
Done